README.md

```diff
@@ -153,9 +188,11 @@ Write Ruby code that will create an instance of a person.
 
 Your answer...
 
+
 ```ruby
-Replace this with your answer
-```
+john = Person.new('John','73')
+
+persons.find_by( age >= 15)```
```

Comment:
In this example, just need to find age = 15, so...

```rb
Person.find_by(age: 15) # also remember that .find_by is a class method, so it needs to be called on Person
```

Although here's a good reference if you want to use conditionals to find something...